### PR TITLE
Basic object reconstruction

### PR DESCRIPTION
**Pull request by T40TDS**
The goal here is, given the minimum number of parameters, reconstruct the object for a query as fully as possible. Basically this is to be used for OOP queries. 

* Doesn't really work for types other than strings yet
* Probably some other issues, this is a first run

### CODE REVIEW

**Review by TrueshotBarrage — APPROVED** [review]
Looks good

File: clientlib/src/DiscreetORM.ts [hunk 1]
```diff
@@ -87,12 +89,35 @@ export class DatabaseORMIO implements DiscreetORMIO {
...
+    }
+
     /** 
      * reconstructObj(entry : DBRowResult) creates an object of type T from a row 
      * entry of that corresponding class database and returns it.
     */
-    reconstructObj<T> (entry : DBRowResult, class_name: string) : T {
-        throw new Error("Not implemented yet")
+    reconstructObj<T> (entry : DBRowResult, class_name: string, column_names: Array<string>) : T {
+        // TODO: Types, and orm_id special handeling, class_name handeling?
```
Comment (TrueshotBarrage) [inline clientlib/src/DiscreetORM.ts]: Should be "handling"

**Comment by T40TDS** [comment]
Builds are failing - I'll fix it

File: clientlib/src/DiscreetORM.ts (hunk 1 above)
Comment (T40TDS) [reply clientlib/src/DiscreetORM.ts]: Yes

**Pull request merged**